src/reducers/index.js

```diff
@@ -12,9 +12,13 @@ const initialState = {
 export default (state = initialState, action) => {
   switch (action.type) {
     case INCREMENT:
-    // Fill in the body of this case
+      return {
+        count: state.count + 1
```

Comment:
Looks awesome! Note that in this instance, it's cool to just return a new object with a count property, but if you had multiple properties on your state you would want to copy in all of the other values as well with the spread operator. Something like `{ ...state, count: state.count + 1 }`